src/workflow_tasks.rs

```diff
@@ -128,7 +134,7 @@ impl WorkflowTaskManager {
             outstanding_activations: Default::default(),
             ready_buffered_wft: Default::default(),
             workflow_activations_update,
-            eviction_policy,
+            cache_manager: RwLock::new(WorkflowCacheManager::new(eviction_policy)),
```

Comment:
I've also realized there's no point in this being an `RwLock` since it's always locked with write. It can be a mutex.